src/hotspot/share/gc/shared/space.hpp

```diff
@@ -541,63 +541,42 @@ class ContiguousSpace: public CompactibleSpace {
   virtual void verify() const;
 };
 
-
-// A dirty card to oop closure that does filtering.
-// It knows how to filter out objects that are outside of the _boundary.
-class FilteringDCTOC : public DirtyCardToOopClosure {
+// A dirty card to oop closure for contiguous spaces (ContiguousSpace and
+// sub-classes). It knows how to filter out objects that are outside of the
+// _boundary.
+//
+// 1. That the actual top of any area in a memory region
+//    contained by the space is bounded by the end of the contiguous
+//    region of the space.
+// 2. That the space is really made up of objects and not just
+//    blocks.
+class ContiguousSpaceDCTOC : public DirtyCardToOopClosure {
 protected:
-  // Override.
+  // Overrides.
```

Comment:
Instead of the `overrides` comment, both methods could be decorated with the `override` specifier.

Reply:
I will do that in another PR for the complete file.